### ISSUE TITLE
Mandatory captcha

### ISSUE THREAD

**Issue by sanscriptum** [matthewn4444/Waifu2x-Image-Saver#5]
Sorry to bother you again.

It seems like waifu2x.booru.pics enforced mandatory captchas on any ip recently so this addon is unusable. You're also required to solve it for every new request which is very vexing.

**Comment by matthewn4444** [comment]
Sorry I have been on hiatus for this project. I actually gave up on using waifu2xbooru and did it locally. Might be able to upload my current version later but it requires you to have a gpu.

**Comment by sanscriptum** [comment]
Yeah I have a gpu that can run waifu locally, but it's just such a hassle to fire it up every time I want to scale something, so it's nice if you have an addon that can do it for you.

**Comment by matthewn4444** [comment]
The current version i am using sometimes breaks during a socket request to do image resizing stuff and then I have to keep enable/disable the addon to continue. Though sometimes and I am not sure if it is the addon or my Firefox but it would freeze after a couple hours. I can try to upload and fix the issue. And i can upload a signed version to install straight away.

**Comment by sanscriptum** [comment]
Please do.
